app/models/comment.rb

```diff
@@ -1,4 +1,17 @@
 class Comment < ApplicationRecord
   belongs_to :author, class_name: 'User'
   belongs_to :post
+
+  after_create :increment_post_comments_counter
+  after_destroy :decrement_post_comments_counter
+
+  private
+
+  def increment_post_comments_counter
+    post.increment!(:comments_count)
+  end
```

Comment:
- This would result in an error because the attribute in the `Post` model is called `comments_counter`, please fix this 🙏